frontend/src/components/core/Sidebar/SidebarNav.tsx

```diff
@@ -135,16 +133,19 @@ const SidebarNav = ({
                       }
                     }}
                   >
-                    {pageIcon && pageIcon.length ? (
+                    {pageIcon && pageIcon.length && (
                       <EmojiIcon size="lg">{pageIcon}</EmojiIcon>
-                    ) : (
-                      <Icon
-                        color="darkenedBgMix100"
-                        content={Description}
-                        size="lg"
-                      />
                     )}
-                    <span>{pageName.replace(/_/g, " ")}</span>
+                    <StyledSidebarLinkText
+                      isActive={pageScriptHash === currentPageScriptHash}
```

Comment:
nit: would also recommend defining a new `isActive` variable since `pageScriptHash === currentPageScriptHash` is computed another time above

Reply:
Makes sense as well, updated in the same commit: https://github.com/streamlit/streamlit/pull/5445/commits/fa74dd75594247e2f49675c29b6fcc3e7c9be126